docs/data-sources/dashboard.md

```diff
@@ -0,0 +1,157 @@
+* `fixed_legend_display_stats` - (Optional) For a chart with a fixed legend, a list of statistics to display in the legend.
+* `fixed_legend_filter_sort` - (Optional) Whether to display `TOP` or `BOTTOM` ranked series in a fixed legend. Valid options are `TOP`, and `BOTTOM`.
+* `fixed_legend_filter_limit` - (Optional) Number of series to include in the fixed legend.
+* `fixed_legend_filter_field` - (Optional) Statistic to use for determining whether a series is displayed on the fixed legend. Valid options are `CURRENT`, `MEAN`, `MEDIAN`, `SUM`, `MIN`, `MAX`, and `COUNT`.
+* `fixed_legend_hide_label` - (Optional) This setting is deprecated.
+* `xmax` - For x-y scatterplots, max value for the X-axis. Set to null for auto.
+* `ymax` - For x-y scatterplots, max value for the Y-axis. Set to null for auto.
+* `xmin` - For x-y scatterplots, min value for the X-axis. Set to null for auto.
+* `ymin` - For x-y scatterplots, min value for the Y-axis. Set to null for auto.
+* `time_based_coloring` - For x-y scatterplots, whether to color more recent points as darker than older points.
+* `sparkline_display_value_type` - For the single stat view, where to display the name of the query or the value of the query. Valid options are `VALUE` or `LABEL`.
+* `sparkline_display_color` - For the single stat view, the color of the displayed text (when not dynamically determined). Values should be in RGBA format.
+* `sparkline_display_vertical_position` - This setting is deprecated.
+* `sparkline_display_horizontal_position` - For the single stat view, the horizontal position of the displayed text. Valid options are `MIDDLE`, `LEFT`, `RIGHT`.
+* `sparkline_display_font_size` - For the single stat view, the font size of the displayed text, in percent.
+* `sparkline_display_prefix` - For the single stat view, a string to add before the displayed text.
+* `sparkline_display_postfix` - For the single stat view, a string to append to the displayed text.
+* `sparkline_size` - For the single stat view, this determines whether the sparkline of the statistic is displayed in the chart. Valid options are `BACKGROUND`, `BOTTOM`, `NONE`.
+* `sparkline_line_color` - For the single stat view, the color of the line. Values should be in RGBA format.
+* `min` - Min value of the Y-axis. Set to null or leave blank for auto.
+* `plain_markdown_content` - The markdown content for a Markdown display, in plain text.
+* `sparkline_fill_color` - For the single stat view, the color of the background fill. Values should be in RGBA format.
+* `sparkline_value_color_map_colors` - For the single stat view, a list of colors that differing query values map to. Must contain one more element than `sparkline_value_color_map_values_v2`. Values should be in RGBA format.
+* `sparkline_value_color_map_values_v2` - For the single stat view, a list of boundaries for mapping different query values to colors.  Must contain one element less than `sparkline_value_color_map_colors`.
```

Comment:
There's double space between the sentences. Same applies to line 139 in this file and to the same chart settings in the dashboards.md file. This is not a big deal, but we usually use a single space after the full stop :).